hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/utils/FrameUtilsTest.java

```diff
@@ -393,6 +395,47 @@ void checkContractRequiredLongZeroTooBig() {
         verify(featureFlags).isAllowCallsToNonContractAccountsEnabled(DEFAULT_CONTRACTS_CONFIG, null);
     }
 
+    @Test
+    void configuredPrecompileShouldBeDisabled() {
+        Configuration DISABLED_PRECOMPILE_CONFIG = HederaTestConfigBuilder.create()
+                .withValue("contracts.precompile.disabled", "2,4")
+                .getOrCreateConfig();
+        givenNonInitialFrame();
+        given(frame.getMessageFrameStack()).willReturn(stack);
+        given(initialFrame.getContextVariable(CONFIG_CONTEXT_VARIABLE)).willReturn(DISABLED_PRECOMPILE_CONFIG);
+
+        var shouldBeFalse = FrameUtils.isPrecompileEnabled(Address.precompiled(2), frame);
+        assertFalse(shouldBeFalse);
+
+        shouldBeFalse = FrameUtils.isPrecompileEnabled(Address.precompiled(4), frame);
+        assertFalse(shouldBeFalse);
+
+        var shouldBeTrue = FrameUtils.isPrecompileEnabled(Address.precompiled(1), frame);
+        assertTrue(shouldBeTrue);
+
+        shouldBeTrue = FrameUtils.isPrecompileEnabled(Address.precompiled(3), frame);
+        assertTrue(shouldBeTrue);
+
+        shouldBeTrue = FrameUtils.isPrecompileEnabled(Address.precompiled(8), frame);
+        assertTrue(shouldBeTrue);
+    }
+
+    @Test
+    void allPrecompileEnabledIfDisabledListEmpty() {
```

Comment:
Q: Is there a simple test to show a revert or a no op if an actual contract execution was done with a disabled precompile call?

Reply:
added